source/_patterns/02-molecules/body/_body.scss

```diff
@@ -54,7 +54,7 @@ $_config: map-merge-by-keys($_config_schemes, base, $_config_schemes, $scheme);
 
 .jcc-body__aside {
   @include at-media(desktop) {
-    @include grid-col(4);
+    @include grid-col(5);
```

Comment:
I'm not sure the intention was to also increase the sidebar width. You should check with Kirsten.

Reply:
@robdecker I updated this to remain the same overall width, but utilize a single offset column on the aside. What do you think of that? I'm mainly trying to avoid a situation where not utilizing 12 cols creates an unintended display.

Reply:
Ah yeah, an offset works too.